images/kwok/build.sh

```diff
@@ -150,6 +170,21 @@ function main() {
     .
 }
 
+function build_with_nerdctl() {
+  local -n build_args=${1}
+  local -n push_images=${2}
```

Comment:
References are very rare and it is better to avoid using it to avoid possible future misunderstandings, and keep the same args for docker and nerdctl.

Reply:
nerdctl does not support specifying multiple images when push in the same time, I have not other better ideas to pass the images to `build_with_nerdctl`.

is there any solutions in your mind?

Reply:
kindle ping @wzshiming

Reply:
Maybe we can split it into two functions `build_with_nerdctl` and `push_with_nerdctl `

Reply:
I changed `--all-platforms` to appending `--platform` because `--all-platforms` may push the previously built images.

but this change makes it become 2 args to pass to `push_with_nerdctl` again...
so I put the push directly on `main`